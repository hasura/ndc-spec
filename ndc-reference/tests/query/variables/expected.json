[
  {
    "rows": [
      {
        "id": 1,
        "title": "The Next 700 Programming Languages"
      }
    ]
  },
  {
    "rows": [
      {
        "id": 2,
        "title": "Why Functional Programming Matters"
      }
    ]
  }
]