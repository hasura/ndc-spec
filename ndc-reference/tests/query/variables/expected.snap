---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/variables/request.json
---
[
  {
    "rows": [
      {
        "id": 1,
        "title": "The Next 700 Programming Languages"
      }
    ]
  },
  {
    "rows": [
      {
        "id": 2,
        "title": "Why Functional Programming Matters"
      }
    ]
  }
]
